<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeDashboard</class>
 <widget class="QMainWindow" name="EmployeeDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EmployeeDashboard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>332</x>
      <y>80</y>
      <width>481</width>
      <height>192</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
    </property>
    <property name="tabKeyNavigation">
     <bool>true</bool>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragDropMode::NoDragDrop</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::TextElideMode::ElideNone</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5Designation">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>270</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Designation">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>270</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Designation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnEmployee">
    <property name="geometry">
     <rect>
      <x>62</x>
      <y>340</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLabel" name="EmpId">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Employee Id</string>
    </property>
   </widget>
   <widget class="QLabel" name="TimerLabel">
    <property name="geometry">
     <rect>
      <x>592</x>
      <y>30</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="CompanyName">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>100</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Company Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backbutton">
    <property name="geometry">
     <rect>
      <x>666</x>
      <y>310</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Back goto Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3EName">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>160</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="JoiningDate">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>220</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Joining Date</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2Cname">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>100</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_empid">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>40</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelcurrentemp">
    <property name="geometry">
     <rect>
      <x>342</x>
      <y>300</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="EmpName">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Employee Name</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>220</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
